src/components/header/Header.tsx

```
@@ -148,25 +155,24 @@ const Header = () => {
-    } else if (userLoadError) {
-      return <button className="h-10 w-10 overflow-hidden rounded-full" onClick={handleProfileClick}>
-                <Image
-                  src={"/abstract-profile.png"}
+    } 
+    return (<button className="h-10 w-10 overflow-hidden rounded-full" onClick={handleProfileClick}>
+              {
+                user?.imageUrl ?
+                <img
+                  src={user.imageUrl}
                   alt="프로필 이미지"
+                  className="h-full w-full object-cover"
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)